recipe/meta.yaml

```diff
@@ -16,20 +18,18 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}.tar.Z  # [not win]
-  fn: {{ name }}.zip  # [win]
-  url: {{ root_url }}/{{ platform_segment }}/packages/{{ name }}.tar.Z  # [not win]
-  url: {{ root_url }}/{{ platform_segment }}/packages/{{ name }}.zip  # [win]
+  url: {{ root_url }}/{{ platform_segment }}/packages/{{ name }}.{{ file_ext }}
   sha256: {{ sha256 }}
   
   patches:
     - 0003_remove_m64_linux.patch  # [linux64 or aarch64]
 
 build:
-  number: 6
+  number: 7
 
 requirements:
   build:
+    - {{ stdlib("c") }}
```

Comment:
@AndrewAnnex This stdlib change is something the linter picked up which is probably worth migrating to anyway irrespective of the aarch64/ppcle64 stuff.